code/modules/projectiles/magazines/flamer.dm

```diff
@@ -119,3 +119,13 @@
 
 	default_ammo = /datum/ammo/water
 	dispenser_type = /obj/structure/reagent_dispensers/watertank
+
+//The engineer pyro bag internal fuel tank
+/obj/item/ammo_magazine/flamer_tank/internal
+	name = "internal fuel tank"
+	desc = "A large fuel tank of ultra thick napthal, a sticky combustable liquid chemical, you shouldn't see this though."
```

Comment:
Napthal?

Reply:
What should it be called? Thats our fuel I think